src/ClientHandle.cpp

```diff
@@ -2075,12 +2078,10 @@ bool cClientHandle::CheckBlockInteractionsRate(void)
 	ASSERT(m_Player != nullptr);
 	ASSERT(m_Player->GetWorld() != nullptr);
 
-	if ((cRoot::Get()->GetServer()->ShouldLimitPlayerBlockChanges()) && (m_NumBlockChangeInteractionsThisTick > MAX_BLOCK_CHANGE_INTERACTIONS))
-	{
-		return false;
-	}
-
-	return true;
+	return !(
+		(cRoot::Get()->GetServer()->ShouldLimitPlayerBlockChanges()) &&
+		(m_NumBlockChangeInteractionsThisTick > MAX_BLOCK_CHANGE_INTERACTIONS)
+	);
```

Comment:
Honestly, I find this a lot less readable than the original version.

Reply:
omg it's xoft hello again

Reply:
Yea I'm still here, but with less time to read the hundreds of notification emails I get.